### PR TITLE
[hi] Localize contribute/style/_index.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/contribute/style/_index.md` to Hindi `content/en/docs/contribute/style/_index.md`

- Completing this feature issue fixes part of :-https://github.com/kubernetes/website/issues/36455

Fixes :- #36510

### CODE REVIEW

File: content/hi/docs/contribute/style/_index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: प्रलेखन शैली अवलोकन
+main_menu: true
+weight: 80
+---
+
+इस खंड के विषय लेखन शैली, सामग्री स्वरूपण, और संगठन, और कुबेरनेट्स प्रलेखन के लिए विशिष्ट ह्यूगो अनुकूलन का उपयोग करना ये सब मार्गदर्शन प्रदान करते हैं।
```
Comment (Babapool) [inline content/hi/docs/contribute/style/_index.md:7]: `Hugo` is a software/framework, it is better not to localize it. Also this way the sentence sounds better, WDYT, thoughts?
```suggestion
इस खंड के विषय लेखन शैली, सामग्री स्वरूपण, और संगठन, और कुबेरनेट्स प्रलेखन के लिए विशिष्ट Hugo अनुकूलन का उपयोग करने पर मार्गदर्शन प्रदान करते हैं।
```

Comment (divya-mohan0209) [reply content/hi/docs/contribute/style/_index.md:7]: + 1 to what @Babapool said.

**Comment by Babapool** [comment]
The changes LGTM.

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

**Comment by divya-mohan0209** [comment]
Taking approval from [this comment](https://github.com/kubernetes/website/pull/36511#issuecomment-1236263356), signing this off for merge.
/approve

**Pull request merged**